src/einsteinpy/tests/test_plotting/test_scattergeodesicplotter.py

```diff
@@ -30,7 +36,10 @@ def test_plot_attractor_is_called_only_once(dummy_data):
 @mock.patch(
     "einsteinpy.plotting.geodesics_scatter.ScatterGeodesicPlotter._plot_attractor"
 )
-def test_plot_calls_plot_attractor(mock_plot_attractor):
+def test_plot_calls_plot_attractor1s(mock_plot_attractor):
```

Comment:
Why renaming?

Reply:
I was testing for the coverage, which functions are not executing, so I added 1,2,3 after the names so it will be easy to check where it happened. I planned to change this once the checks were passed.